### PR TITLE
DDSDK-41 Theme kickstart

### PR DESCRIPTION
**Pull request by troelslenda**

### CODE REVIEW

**Comment by mian3010** [comment]
Retest this please

**Comment by reload-branching-check** [comment]
**Reload Commit Checker**
The following commits does not correctly reference a Jira issue:

|SHA|Message|Author|Date|
|---|-------|------|----|
|04d7e273c716a1b9d718984413046ffb8f6800a0|Creating Mungo theme|Troels Lenda (troelslenda@gmail.com)|2016-09-13T06:19:35Z|
|d01be81b37b8b6b4f3b12bf88e50c75510686544|Providing standard node modules to build project.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T07:57:25Z|
|e813ea5af0a344f41e4a84cd3b0842d320b5181e|Prepare theme_assets|Troels Lenda (troelslenda@gmail.com)|2016-09-13T07:57:45Z|
|bc667dc42542ba78a74610b6900d783e907f66e8|Build assets for theme and correct error where variables weren't set.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T08:35:16Z|
|ca30c3b63415445b64c5bfcc2248cd41c249dedb|Use Gulp generated styles in theme.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T08:35:44Z|
|661de17e5e43674984902529c2e5ac7888d2c54a|Ensure that gulp isen't exited on sass compile error.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T08:50:02Z|
|84561ec6b2e411bf5c96b3d50decbbd7eac405c0|Better naming|Troels Lenda (troelslenda@gmail.com)|2016-09-13T08:50:16Z|
|2aed09c254a05cd15089e8e33f31cd0f28c479f7|Extending gulp config with:<br />-javascript assets copying and linting.<br />-include Sass Bourbon.<br />-cleaning of removed files.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T10:11:35Z|
|24f4fe9e722ecd85a18d15270edab40a33b73c83|Updating theme with correct inheritance and dependencies|Troels Lenda (troelslenda@gmail.com)|2016-09-13T10:12:27Z|
|68f5ef41070e82a5aedbdd7b0957630f2b7e3830|Use bootstrap from NPM.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T10:58:08Z|
|99162fa2d1a75a5d5f20d273b617fd070a3f7a98|Updating configuration to enable Mungo theme by default.|Troels Lenda (troelslenda@gmail.com)|2016-09-13T12:20:15Z|

File: web/themes/custom/mungo/mungo.info.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Mungo
+type: theme
+base theme: classy
+description: 'Mungo is a Mongoose with courrage and speed as its main perks.'
```
Comment (danquah) [inline web/themes/custom/mungo/mungo.info.yml]: courage

**Comment by danquah** [comment]
Review OK - overvej lige stavefejlen

**Pull request merged**